app/models/car.rb

```diff
@@ -0,0 +1,10 @@
+class Car < ApplicationRecord
+  has_many :reservations, dependent: :destroy
+
+  validates :name, presence: true
+  validates :make, presence: true
+  validates :description, presence: true
+  validates :cost, presence: true, numericality: { greater_than: 0 }
+  validates :speed, presence: true, numericality: { greater_than: 0 }
+  # validates :color, presence: true
```

Comment:
- [ ] Kindly, remove this old piece of commented code in order to improve the repo's cleanliness. 👍